src/Main.java

```diff
@@ -1,5 +1,63 @@
 public class Main {
     public static void main(String[] args) {
         System.out.println("Home");
+        var dog = 8;
+        System.out.println(dog);
+        var cat = 3.6;
+        System.out.println(cat);
+        var paper = 763789;
+        System.out.println(paper);
+        // 1 задача сверху
+        dog = dog + 4;
+        System.out.println(dog);
+        cat = cat + 4;
+        System.out.println(cat);
+        paper = paper + 4;
+        System.out.println(paper);
+        // 2 задача сверху
+        double dog1 = dog - 3.5;
+        System.out.println(dog1);
+        cat = cat - 1.6;
+        System.out.println(cat);
+        paper = paper - 7639;
+        System.out.println(paper);
+        // 3 задача сверху
+        var friend = 19;
+        System.out.println(friend);
+        friend = friend + 2;
+        System.out.println(friend);
+        friend = friend / 7;
+        System.out.println(friend);
+        // 4 задача сверху
+        var frog = 3.5;
+        System.out.println(frog);
+        frog = frog * 10;
+        System.out.println(frog);
+        frog = frog / 3.5;
+        System.out.println(frog);
+        frog = frog + 4;
+        System.out.println(frog);
+        // 5 задача сверху
+        var weight0neBoxer = 78.2;
+        var weightTwoBoxer = 82.7;
+        var totalWeight = weight0neBoxer + weightTwoBoxer;
+        System.out.println("Общий вес боксёров " + totalWeight + " кг!");
+        var differenceWeight = weightTwoBoxer - weight0neBoxer;
+        System.out.println( " Разница в вессе " + differenceWeight + " кг!");
+        // 6 задача сверху
+        var weightSubtraction = weightTwoBoxer - weight0neBoxer;
+        System.out.println(" Разница в вессе " + weightSubtraction + " кг!");
+        var divisionWeight = weightTwoBoxer % weight0neBoxer;
+        System.out.println(" Разница в вессе " + divisionWeight + " кг!");
+        // 7 задача сверху
+        var totalHours = 640;
+        var normHours = 8;
+        var staff = totalHours / normHours;
+        System.out.println("Всего работников в компании " + staff );
+        staff = staff + 94;
+        var newHours =  totalHours / staff;
```

Comment:
чтобы узнать общее число часов надо общее число сотрудников умножить на кол-ва часов на одного сотрудника
```suggestion
        totalHours  =  normHours * staff;
```